common/history/characters/0_dvg_spain.txt

```diff
@@ -6,10 +6,10 @@
 			ruler = yes
 			noble = yes
 			birth_date = 1830.10.10
-			interest_group = ig_petty_bourgeoisie
+			interest_group = ig_landowners
 			ideology = ideology_royalist
 			traits = {
-				persistent
+				jingoist
```

Comment:
[22:23:01][pdx_persistent_reader.cpp:252]: Error: "Unexpected token: SCO, near line: 1035" in file: "common/flag_definitions/dvg_flag_definitions.txt" near line: 1044
[22:23:02][character_templates_utils.cpp:29]: Invalid Character trait provided jingoist in  file: common/history/characters/0_dvg_spain.txt line: 3
[22:23:02][jomini_effect.cpp:128]: PostValidate of effect 'create_character' returned false at  file: common/history/characters/0_dvg_spain.txt line: 3